### ISSUE TITLE
Question about the application 

### ISSUE THREAD

**Issue by TDH2503** [landaire/wows-toolkit#7]
![image](https://github.com/user-attachments/assets/adf5757a-0a8c-47cb-b1c2-e2f3b47e56b1)
my window pc antivirus program detected this from your file: C:\wows\World_of_Warships\wows_toolkit.exe , but after i closed down the antivirus program it works perfectly normal.
![Screenshot 2024-11-13 010005](https://github.com/user-attachments/assets/cfd7fd0a-2933-4cd5-bc53-6413fbdb491d)
so is it ok just click allow it ? since its worked perfectly normal after i closed down my antivirus program temporary which i assume it is fine.
i have very limited knowledge in programming stuffs as my profession is biology  so please forgive my ignorance if i missed something else important while being common sense to you.  It would be plesant if you can interpret the situation i encountered and give an solution to it.

**Comment by landaire** [comment]
Did you build it manually? Or did you use my pre-compiled builds from the [releases page](https://github.com/landaire/wows-toolkit/releases). If you built yourself that's actually pretty impressive!

For some reason Windows does detect the tool as a virus if you build yourself. I have no idea why. The pre-built application is using a different compiler backend which for some reason Windows Defender does not flag as a virus.

There are a couple other factors too, such as the application not being digitally signed which gives it a higher trust score with the antivirus, and the application not being widely used by a lot of folks. I can look into getting a code signing pipeline set up so that hopefully it makes the application more trusted in the eyes of antivirus.

**tl;dr** f you download the application from this repository it is safe to use and the antivirus warning is a false positive.
